.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - 8
+script:
+  - node_modules/bs-platform/bin/bsb -make-world
```

Comment:
My cursory search implies that this should be added to the script:
```yaml
before_script:
  - npm install
```
Is this correct?